src/test/java/com/wikia/webdriver/testcases/mercurytests/PortableInfoboxTests.java

```diff
@@ -0,0 +1,127 @@
+  public void MercuryPortableInfoboxTest_001_ElementsVisibility() {
+    PortableInfoboxObject infobox = new PortableInfoboxObject(driver);
+    infobox.openMercuryArticleByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+
+    // Before infobox expanding
+    infobox
+        .isMainImageVisible()
+        .isTitleOverImageVisible()
+        .isImageInTitleNotVisible()
+        .isImageInTabberVisible()
+        .isImageCaptionInTabberVisible();
+
+    // After infobox expanding
+    infobox
+        .clickExpandButton()
+        .isVideoVisible()
+        .isVideoCaptionVisible()
+        .areDataLabelsVisible()
+        .areDataValuesVisible()
+        .areLinksVisible()
+        .areReferencesVisible()
+        .areHeadersVisible()
+        .areUnorderedListsVisible()
+        .areOrderedListsVisible()
+        .areUnorderedListAndDataValuesMarginEqual()
+        .areOrderedListAndDataValuesMarginEqual();
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_002")
+  @Execute(onWikia = "mercuryautomationtesting")
+  public void MercuryPortableInfoboxTest_002_CollapsingMethods() {
+    PortableInfoboxObject infobox = new PortableInfoboxObject(driver);
+    infobox.openMercuryArticleByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+
+    // expand by clicking button
+    infobox
+        .isInfoboxCollapsed()
+        .clickExpandButton()
+        .isInfoboxExpanded()
+        .clickExpandButton()
+        .isInfoboxCollapsed();
+
+    // expand by tapping Infobox's content
+    infobox
+        .tapInfoboxContent()
+        .isInfoboxExpanded()
+        .tapInfoboxContent()
+        .isInfoboxCollapsed();
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_003")
+  @Execute(onWikia = "mercuryautomationtesting")
+  public void MercuryPortableInfoboxTest_003_ExternalLinkRedirection() {
+    PortableInfoboxObject infobox = new PortableInfoboxObject(driver);
+    infobox.openMercuryArticleByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+
+    String externalLinkName = infobox
+        .clickExpandButton()
+        .areLinksVisible()
+        .getExternalLinkName(0);
+
+    String externalURL = infobox
+        .clickExternalLink(0)
+        .getUrlFromExternalLinkaAfterPageIsLoaded();
+
+    infobox.isExternalLinkLabelInURL(externalLinkName, externalURL);
+  }
+
+  @Test(groups = "MercuryPortableInfoboxTest_004")
+  @Execute(onWikia = "mercuryautomationtesting")
+  @RelatedIssue(issueID = "DAT-3085")
+  public void MercuryPortableInfoboxTest_004_ImageAndVideoOpensInLightbox() {
+    PortableInfoboxObject infobox = new PortableInfoboxObject(driver);
+    infobox.openMercuryArticleByNameWithCbAndNoAds(wikiURL, MercurySubpages.INFOBOX_1);
+
+    // Check image
+    infobox
+        .clickMainImage()
+        .isLightboxOpened()
+        .closeLightbox()
+        .isTitleOverImageVisible();
+
+    // Check video
+    infobox
+        // TODO: uncomment after related issue is done
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)